lib/ansible/module_utils/aws.py

```diff
@@ -0,0 +1,111 @@
+# This code is part of Ansible, but is an independent component.
```

Comment:
most of this already exists in ec2.py (badly named, but that is used by most aws modules)